imagick/statistic.go

```diff
@@ -0,0 +1,25 @@
+package imagick
+
+/*
+#include <MagickCore/MagickCore.h>
+*/
+import "C"
+import "runtime"
+
+// GetImageEntropy returns the entropy of one or more image channels.
+func (mw *MagickWand) GetImageEntropy() (entropy float64, err error) {
```

Comment:
This is just a nitpick, but the existing code doesn't use named return value :-)

Reply:
CI testing was broken for this project and I just fixed it. So if you want to make this small change, it will trigger a new CI build.

Reply:
@justinfx I saw several cases in which named return values are used, such as `GetResolution`, `CompareImages`, and `GetImageBluePrimary`. I'm willing to make the change, but could you let me know why and what are the conventions/principles to follow? I may add more statistic functions in the future :)

Reply:
I apparently didn't look at enough of the existing code as a reference :-) The original author apparently used it, and I assumed I wasn't using it going forward. Seeing as it is mixed, I guess don't worry about it. 
Can you just add yourself to the top level CREDITS file, so that it triggers the CI at least?

Reply:
I've updated CREDITS and CI got triggered. Please check :)